python/conversion/pocolog.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
+
+# Pocolog bindings
+from pocolog_pybind import *
+
+import tqdm
+import warnings
+import numpy as np
+
+from data.format import Events
+
+class EventAccumulator:
```

Comment:
This class is duplicate code from: https://github.com/uzh-rpg/e2calib/blob/cea480c80822f0b91a851609166d6c71cf341e5f/python/conversion/ros.py#L8

We should have a separate file and import the class

Reply:
This is assuming ros EventAccumulator and pocolog EventAccumulator are the same. It is true now there are the same but it might change in the future. right?

Reply:
Atm they are the same and we can reduce code duplication. We could still change back later if needed

Reply:
I just tried to merge them but they're actually different. Please have a look at my comment.